transcom/mymove
# B 17842 New Department Indicators on Orders View/Edit Page

Agility Ticket: [B-17842](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A863767)

## Summary

This adds "57 Air Force and Space Force", "96 Army Corps of Engineers", and "97 Office of Secretary of Defense and all Defense Agencies" to the list of options in the "Department Indicators" dropdown menu in the "View/Edit Orders" page that the TOO and SC have access to.

### How to test

1. (If needed) Create a move and upload orders
2. Login as a SC/TOO and access the move page
3. Click "Edit Order" in the "Orders" module
4. In the department indicators dropdown, the above options should now be available as options.
5. Change to one of the new options and click "Save", the TOO/SC summary screen should now show the new Dept. in the "Orders" module.

### Backend


## Screenshots
![Screenshot 2023-11-15 at 5 25 14 PM](https://github.com/transcom/mymove/assets/146959895/7385d2fb-77eb-4204-ad7a-be6f04bfd4bc)
![Screenshot 2023-11-15 at 5 25 40 PM](https://github.com/transcom/mymove/assets/146959895/fa88b45f-ea64-43f9-8776-9110d90a2e01)
![Screenshot 2023-11-15 at 5 26 07 PM](https://github.com/transcom/mymove/assets/146959895/96ea4704-d7d5-4d39-95c4-9d009ad3a3e3)

Note: This PR is a draft until our team receives clarification regarding the naming of the 97 Dept. name.
